sklearn/model_selection/_split.py

```diff
@@ -76,7 +76,7 @@ def split(self, X, y=None, groups=None):
             Group labels for the samples used while splitting the dataset into
             train/test set.
 
-        Returns
+        Yields
         -------
```

Comment:
You need to remove the last `-` to match the section length.

Reply:
👍